logging/src/posix/logging.cc

```diff
@@ -6,7 +6,8 @@ using namespace std;
 
 namespace leatherman { namespace logging {
 
-    void colorize(ostream& dst, log_level level)
+    template <typename StreamT>
+    void colorize_impl(StreamT& strm, log_level level)
```

Comment:
wow, so formatting_ostream really doesn't inherit from ostream :( That makes me super sad, especially given its name.

Template seems fine here in that case.